InputOutput.py

```diff
@@ -236,7 +236,8 @@ def readInPedigreeFromInputs(pedigree, args, genotypes=True, haps=False, reads=F
     
     aapfile = getattr(args, "alt_allele_prob_file", None)
     if aapfile is not None:
-        pedigree.readInAAP(aapfile)
```

Comment:
@RosCraddock I am not versed/familiar with `pedigree.readInAAP()`. Do we pass in a file or `aap`?

Reply:
@gregorgorjanc This is a new function. Here, we pass in a file path through aap. This file contains the user-input alternative allele frequencies for each metafounder.